lib/arjdbc/jdbc/adapter.rb

```diff
@@ -289,12 +289,14 @@ def active?
 
       # @override
       def reconnect!
+        super
```

Comment:
so this actually helped a piece of test passing?
... not sure if it plays nicely with JNDI - they used to have some weird assumptions (for mostly embed adapters)